CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_quickbooks v0.9.1
+[PR #93](https://github.com/fivetran/dbt_quickbooks/pull/93) includes the following updates:
+## Bug Fixes
+- Adjusted the purchase amount totals within the `int_quickbooks__purchase_transactions` model to factor in credits when calculating purchase amounts. 
```

Comment:
Add an empty line to line 5 to separate out the release notes

Reply:
Updated.